2023/README.md

```diff
@@ -1,20 +1,284 @@
 # Interop 2023
 
-Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and end users. It's part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines.
+Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and end users. This effort is part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines (including Apple, Bocoup, Google, Igalia, Microsoft, and Mozilla).
 
-Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the Interop 2023 Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the Interop team to work on throughout the year.
+The process we went through to arrive at our priority list for Interop 2023 is available in the [planning process](planning-process.md) document.
```

Comment:
When https://github.com/web-platform-tests/interop/pull/268 is merged this will be a better summary:

```suggestion
The process we went through to arrive at our priority list for Interop 2023 is available in the [proposal selection summary](./proposal-selection.md).
```